lib/crpc/server.go

```diff
@@ -300,31 +352,32 @@ func isValidMethod(method, version string) bool {
 // defined above, or the presence of the schema doesn't match the presence
 // of the input argument, Register will panic. This function is not thread safe
 // and must be run in serial if called multiple times.
-func (s *Server) Register(method, version string, schema gojsonschema.JSONLoader, fnR interface{}, mw ...MiddlewareFunc) {
-	if fnR == nil {
-		s.RegisterFunc(method, version, schema, nil, mw...)
-
-		return
-	}
```

Comment:
Why would we ever register a nil handler? I can't see this being used anywhere...